hedera-node/src/main/java/com/hedera/services/tokens/TokenStore.java

```diff
@@ -58,7 +58,7 @@
 
```

Comment:
I think that you should add a new method in the interface `boolean nameExists(String name)` and implement it in the `HederaTokenStore`